natiami/assignment3/Dictionary.cpp

```diff
@@ -0,0 +1,18 @@
+#include "Dictionary.h"
+
+Dictionary::Dictionary(){
+	this->words = set<string>();

```

Comment:
You don't have to use the `this` qualifier whenever accessing a class field.
Just `words` works too.

Some prefer to always use `this`, some prefer to use it sometimes (e.g. to distinguish from a function parameter with a similar name), and some never use it.
That's a matter of style and personal preference, you can choose the style which you thinks makes the code most clear and readable (or alternatively, the style that you are used to from previous works, that's ok too ;) )